studio.py

```diff
@@ -61,17 +61,35 @@ def _digest_file(self, file, algorithm):
             print("hashed %s with %s to be %s" % (name, algorithm, h.hexdigest()))
             return h.hexdigest()
 
-    def replaying(self):
-        with PROXYS[self.proxy](path=self.path, mode="replay") as proxy_service:
-            app_service = APPS[self.app](proxy_service, self.certutil)
-            app_service.start(self.url)
+    def replaying(self, site= None, screen_shot=False):
```

Comment:
Please run black over your patch before pushing to GitHub. You can install this as a precommit hook. It saves time in reviews so we don't need to discuss issues such as the extra space in `site= None`. You can probably also install black in your IDE.